lib/modules/nightMode.js

```diff
@@ -38,27 +39,34 @@ module.options = {
 		advanced: true,
 	},
 	automaticNightMode: {
-		type: 'boolean',
+		type: 'enum',
 		value: false,
+		values: [{
+			name: 'nightModeAutomaticNightModeNone',
+			value: 'none',
+		}, {
+			name: 'nightModeAutomaticNightModeAutomatic',
+			value: 'automatic',
+		}, {
+			name: 'nightModeAutomaticNightModeUser',
+			value: 'user',
+		}],
 		description: 'nightModeAutomaticNightModeDesc',
 	},
 	nightModeStart: {
 		type: 'text',
 		value: '20:00',
 		description: 'nightModeNightModeStartDesc',
-		dependsOn: 'automaticNightMode',
```

Comment:
Would be cool if we could write something like `dependsOn: ['automaticNightMode', (value) => value === 'user']`